two_factor/templates/two_factor/_wizard_forms.html

```diff
@@ -2,3 +2,5 @@
   {{ wizard.management_form }}
   {{ wizard.form }}
 </table>
+{# Add empty paragraph to add some space below the table #}
+<p></p>
```

Comment:
If there needs to be extra space, then you need to add margin to the table. I think Boostrap has a class for that.

Reply:
Bootstrap 4 has, Bootstrap 3 didn't ;-), that's why I added the paragraph-hack. Now updating the branch.